neo4j/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM neo4j:4.2.5
+FROM neo4j:4.2.11
```

Comment:
Since some work has gone into choosing a newer Docker imagine, why not go all the way to the latest `4.3.7`? It would same some time to not go through all the versions incrementally, but perhaps you want a dedicated issue for it?

Reply:
I should have made a comment about this but I didn't get the 4.3.x versions to work locally. I didn't try that much too look into it to be fair. For now this means hopefully less surprises as well since we are staying on 4.2.x.